### PR TITLE
Add bewCloud

### PR DESCRIPTION
**Pull request by BrunoBernardino**
Thanks for maintaining this list, I initially made a PR too early at https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/691, just for reference.

---

Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/bewcloud.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - Docker
+  - Javascript
+tags:
+  - File Transfer & Synchronization
+  - Groupware
+  - Feed Readers
+  - Knowledge Management Tools
+  - Note-taking & Editors
+  - Photo and Video Galleries
+depends_3rdparty: false
```
Comment (Ki-er) [inline software/bewcloud.yml:19]:
```suggestion

```

File: software/bewcloud.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+website_url: "https://bewcloud.com"
+source_code_url: "https://github.com/bewcloud/bewcloud"
+description: "A simpler alternative to Nextcloud and ownCloud's RSS Reader, File Sharing + Sync, Notes, and Photos."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Deno
+  - Nodejs
+  - Docker
+  - Javascript
```
Comment (Ki-er) [inline software/bewcloud.yml:11]: With  Nodejs being present this isnt needed.
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]

**Review by kokomo123 — APPROVED** [review]
LGTM, shouldn't be anything wrong. Pending further thoughts from collaborators. Thank you for your contribution!

File: software/bewcloud.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - AGPL-3.0
+platforms:
+  - Deno
+  - Nodejs
+  - Docker
+tags:
+  - File Transfer & Synchronization
+  - Groupware
+  - Feed Readers
+  - Knowledge Management Tools
```
Comment (nodiscc) [inline software/bewcloud.yml:15]:
```suggestion
```

does not apply, if this refers to the notes functionality, this is covered by `Note-taking & Editors`

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/bewcloud.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
+name: "bewCloud"
+website_url: "https://bewcloud.com"
+source_code_url: "https://github.com/bewcloud/bewcloud"
+description: "A simpler alternative to Nextcloud and ownCloud's RSS Reader, File Sharing + Sync, Notes, and Photos."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Deno
+  - Nodejs
+  - Docker
```
Comment (nodiscc) [inline software/bewcloud.yml:10]:
```suggestion
platforms:
  - Docker
```

Docker is the only supported deployment method

>  Values for platform are the main server-side requirements for the software. Don't include frameworks or specific dialects.

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**